src/tests/baseservices/RuntimeConfiguration/TestConfig.cs

```diff
@@ -156,4 +159,13 @@ static string GetCorerunPath()
         }
         return Path.Combine(Environment.GetEnvironmentVariable("CORE_ROOT"), corerunName);
     }
-}
\ No newline at end of file
+
+    static void ClearEnvironmentVariable(string name)
+    {
+        string value = Environment.GetEnvironmentVariable(name);
+        if (!string.IsNullOrEmpty(value))
+        {
+            Environment.SetEnvironmentVariable(name, null);
```

Comment:
After @AaronRobinsonMSFT 's change to enable the DOTNET_ prefix, I believe the CLR reads both the DOTNET and COMPlus prefixes case-insensitively on Linux. (Used to be case sensitive)
https://github.com/dotnet/runtime/pull/50507/files#diff-ca28f8b9cc75904774104e92e308462d908149b3e95288e37acd5b719bf6e0e9R643

This makes it difficult to clear them on Linux, since you don't know the casing. I guess you have to enumerate all variables, pick out any with these prefixes case insensitively, then use those names to clear them?

Reply:
> The test infrastructure always seems to use the `COMPlus_` prefix

(separate from this PR) At some point, we can update the test infra to use `DOTNET_` prefixes.